providers/ard/ard.go

```diff
@@ -0,0 +1,25 @@
+package ard
+
+import (
+	"fmt"
+	"net/url"
+
+	"github.com/zquestz/s/providers"
+)
+
+func init() {
+	providers.AddProvider("ard", &Provider{})
```

Comment:
Perhaps we should use the whole name? `ardmediathek`?

There is already tab and partial provider name completion. So you could still type -p ard if you want. =)